notifications/hub.go

```diff
@@ -36,6 +36,11 @@ func NewHub() *Hub {
 	}
 }
 
+// SubscriberCount returns the number of subscribers
+func (h *Hub) SubscriberCount() int {
+	return len(h.subscribers)
```

Comment:
you need go grab the lock here, as it's a public method that looks at `h.subscribers`